perllib/FixMyStreet/Roles/CobrandSLWP.pm

```diff
@@ -203,15 +204,24 @@ sub available_permissions {
 sub clear_cached_lookups_property {
     my ($self, $id) = @_;
 
-    my $key = $self->council_url . ":echo:look_up_property:$id";
-    delete $self->{c}->session->{$key};
-    $key = $self->council_url . ":echo:bin_services_for_address:$id";
-    delete $self->{c}->session->{$key};
+    foreach my $key (
+        $self->council_url . ":echo:look_up_property:$id",
+        $self->council_url . ":echo:bin_services_for_address:$id",
+        $self->council_url . ":echo:available_bulky_slots:earlier:$id",
+        $self->council_url . ":echo:available_bulky_slots:later:$id",
+        $self->council_url . ":echo:bulky_event_guid:$id",
```

Comment:
Should the last three of these be in a separate `clear_cached_lookups_bulky_slots` function - that's called from the Bulky controller in one place.